StripeCore/StripeCore/Source/Analytics/AnalyticsClientV2.swift

```diff
@@ -134,6 +134,9 @@ extension AnalyticsClientV2Protocol {
             "install": InstallMethod.current.rawValue,
             "app_bundle_id": Bundle.stp_applicationBundleId() ?? "",
         ]
+        if let deviceId = UIDevice.current.identifierForVendor?.uuidString {
```

Comment:
Is there a mapping somewhere of UUIDs to device model names? I'm familiar with [this list of device identifiers](https://gist.github.com/adamawolf/3048717), but it wouldn't be the same as `UIDevice.current.identifierForVendor?.uuidString`

Reply:
This is a unique identifier for the specific device/app install, so it does not map to any device types. This means that if one device had multiple apps with the SDK integrated then the identifier will be different (unless the apps were distributed by the same developer).

Reply:
The Vendor ID is unrelated to the device model and is a per-device x per-app hash. When the user installs the app, the system vends a unique UUID identifier:
https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor/

If you want to know the device model, then I think you probably want to use `device_type` which is populated by this:
https://github.com/stripe/stripe-ios/blob/1494f048/StripeCore/StripeCore/Source/Helpers/STPDeviceUtils.swift